dotnetv3/Glue/Scenarios/Glue_Basics/GlueBasics.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+// snippet-start:[Glue.dotnetv3.GlueBasics.Main]
+
+using Amazon.Glue.Model;
+
+namespace GlueBasics;
+
+public class GlueBasics
+{
+    private static ILogger logger = null!;
+    private static IConfiguration _configuration;
+
+    static async Task Main(string[] args)
+    {
+        // Set up dependency injection for the Amazon service.
+        using var host = Host.CreateDefaultBuilder(args)
+            .ConfigureLogging(logging =>
+                logging.AddFilter("System", LogLevel.Debug)
+                    .AddFilter<DebugLoggerProvider>("Microsoft", LogLevel.Information)
+                    .AddFilter<ConsoleLoggerProvider>("Microsoft", LogLevel.Trace))
+            .ConfigureServices((_, services) =>
+            services.AddAWSService<IAmazonGlue>()
+            .AddTransient<GlueWrapper>()
+            .AddTransient<UiWrapper>()
+            )
+            .Build();
+
+        logger = LoggerFactory.Create(builder => { builder.AddConsole(); })
+        .CreateLogger<GlueBasics>();
+
+        _configuration = new ConfigurationBuilder()
+            .SetBasePath(Directory.GetCurrentDirectory())
+            .AddJsonFile("settings.json") // Load test settings from .json file.
```

Comment:
remove the word "test", these are settings for the scenario.

Reply:
removed